miniconf/load_site_data.py

```diff
@@ -26,23 +26,53 @@ def load_site_data(
 
     NOTE: site_data[filename][field]
     """
+    registered_sitedata = {
```

Comment:
Use hard-coded list so we don't load all files under `sitedata_acl2020`.
This means less surprise as we have been adding different files to the `sitedata_acl2020` folder.